Assets/Scripts/Combat/CombatController.cs

```diff
@@ -164,7 +211,7 @@ public void EndAction(int spentActionPoints)
             isTurn = false;
             return;
         }
-        FindSelectableTiles();
+        FindSelectableBasicTiles();
```

Comment:
I would get rid of the ClearVisitedTiles (7 lines above) and incorporate it as 1st line in the FindSelectableTiles method.